README.md

```diff
@@ -152,6 +152,41 @@ JSON output:
 }
 ```
 
+#### Custom Search With Filters <!-- omit in toc -->
+
+If you want to enable filtering, you must add your attributes to the `filterableAttributes` index setting.
+
+```js
+await index.updateFilterableAttributes(['id', 'genres']);
+```
+
+You only need to perform this operation once.
+
+Note that MeiliSearch will rebuild your index whenever you update `filterableAttributes`.
+Depending on the size of your dataset, this might take time. You can track the process using the [update status](https://docs.meilisearch.com/reference/api/tasks.html#get-an-update-status).
```

Comment:
```suggestion
Depending on the size of your dataset, this might take time. You can track the process using the [task](https://docs.meilisearch.com/reference/api/tasks.html#get-an-update-status).
```
Or maybe task status if you prefer
```suggestion
Depending on the size of your dataset, this might take time. You can track the process using the [update status](https://docs.meilisearch.com/reference/api/tasks.html#get-an-update-status).
```

Reply:
Thanks!